erpnext/manufacturing/report/production_order_stock_report/production_order_stock_report.py

```diff
@@ -0,0 +1,135 @@
+# Copyright (c) 2017, Velometro Mobility Inc and contributors
+# For license information, please see license.txt
+
+from __future__ import unicode_literals
+from frappe.utils import flt, cint
+import frappe
+
+def execute(filters=None):
+	prod_list = get_production_orders()
+	data = get_item_list( prod_list, filters)
+	columns = get_columns()
+	return columns, data
+	
+def get_item_list(prod_list, filters):
+	out = []
+	
+	low_price_data = []
+	low_supplier = []
+	
+	# Get the default supplier of suppliers
+	#Add a row for each item/qty
+	for root in prod_list:
+		bom = frappe.db.get_value("Production Order", root.name,"bom_no")
+		warehouse = frappe.db.get_value("Production Order", root.name,"source_warehouse")
+		warehouse = "Stores - VMI"
+		desc = frappe.db.get_value("BOM", bom, "description")
+		qty = frappe.db.get_value("Production Order", root.name,"qty")
+		
+		item_list = frappe.db.sql("""SELECT 
+				bom_item.item_code as item_code,
+				SUM(ifnull(ledger.actual_qty * conf_ledger.conversion_factor,0))/(bom_item.qty * conf_item.conversion_factor) as build_qty
```

Comment:
Why used `sum`?

Again `conversion_factor` is not needed. The qty in both "BOM Item" and "Bin" table are in stock uom, hence no need to join with "UOM Conversion Detail" table.